app/tests/unit/routes/v1/form.spec.js

```diff
@@ -900,7 +901,7 @@ describe(`${basePath}/:formId/export/fields`, () => {
 });
 
 describe(`${basePath}/:formId/options`, () => {
-  const path = `${basePath}/:formId/options`;
+  const path = `${basePath}/${uuidv4()}/options`;
```

Comment:
This is a bit of a warning flag that the _route_ tests are actually testing the _controller_. That's a whole other topic of testing that needs to be dealt with!